### ISSUE TITLE
Orca Screen Reader Will Not Read Category List in Main Window

### ISSUE THREAD

**Issue by mcb2003** [ubuntu-mate/mate-tweak#38]
I'm _not_ experienced enough to know why: all I know is that if I open mate tweak, orca, the linux screen reader, won't read any of the categories like windows, interface, etc on the left of the window any more and what's more,  I haven't found a way to switch between them. This I believe started after 16.04, as I haven't had this problem with the LTS.

I made a video discussing it, you can find it at:
https://youtu.be/m9wi17wkhGw

**Comment by alexarnaud** [comment]
Hello,

I can confirm the issue on Debian Sid with Mate Tweak 18.04.16-1.

@flexiondotorg Do you plan to figure out this issue? It's not an accessibility issue but a keyboard issue. The tabs are not shown as tab and it's not possible to go to the tabs list with the keyboard.

Best regards,
Alex.

**Comment by dahunt1962** [comment]
User cannot navigate these tabs either with single-keystroke commands or Orca screen reader's mouse emulation/flat review. Category chooser reads only as "layered pane", with no content.

**Comment by alexarnaud** [comment]
@dahunt1962 I understand. I'm currently looking what's happened.

Best regards,
Alex.

**Comment by alexarnaud** [comment]
I've worked a little bit to find how it works with the help of @shindere 

The sidebar is constructed on the file mate-tweak at line 1413 on master.

FYI, it's the first time I'm looking at the source code to try to figure out an accessibility issue and I'm not familiar with the GTK toolkit.

@clefebvre @monsta @flexiondotorg Do you have any idea why the sidebar is not keyboard navigable and why there are no accessibility label added to the image ? Do you think we could imagine to add an accessible name to all the image to make them accessible ?

Best regards,
Alex.

**Comment by shindere** [comment]
To add so what @alexarnaud explained, I think we also had the
understanding that most of the interface-related objects were built
using Glade and thus did appear in the .ui file, whereas this SidePage
was built directly in the code and did not appear in the .ui file. Is
that correct?

It also seemed that the objects which are accessible and
keyboard-navigable are precisely those defined with Glade. We wondered
why it is so and why the sidebar is not defined within Glade.

**Comment by monsta** [comment]
Could be caused by this: https://github.com/ubuntu-mate/mate-tweak/commit/d75f0cd3515188d54a8c929b486400d4491d5932#diff-ce5869a9c0f08875d1603efbf5b1c980R29

**Comment by monsta** [comment]
Ok, keyboard focus is back in https://github.com/ubuntu-mate/mate-tweak/pull/47, please test if it helps with Orca too.

**Comment by flexiondotorg** [comment]
Fixed via https://github.com/ubuntu-mate/mate-tweak/pull/47/commits/ab2c21314c6471212dcd3a29c056cb2edb2bce29
